llvm/lib/Target/X86/lab3/kozyreva_ekaterina/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(PluginName X86KozyrevaPass)
+file(GLOB_RECURSE ALL_SOURCE_FILES *.cpp *.h)
+add_llvm_pass_plugin(${PluginName}
+        ${ALL_SOURCE_FILES}
+        DEPENDS
+        intrinsics_gen
+        X86
+        BUILDTREE_ONLY)
+
+target_include_directories(${PluginName} PUBLIC ${PATH_TO_X86})
```

Comment:
Please add current plugin into dependencies for tests

Reply:
I did it